docs/litmus_scenarios.md

```diff
@@ -0,0 +1,18 @@
+### Litmus Scenarios
+Kraken consumes [Litmus](https://github.com/litmuschaos/litmus) under the hood for some infrastructure, pod, and node scenarios 
+
+#### Litmus chaos scenarios
+There are 3 yaml files needed to properly execute each litmus scenario. Below is a description of each file needed
```

Comment:
ChaosResult is not needed to be created - it will be created by litmus itself and updated by litmus.
The only responsibility will be to inspect the reported status and to clear the resource.